ActionFiles/V1/TGMS_Bookmarks.act

```diff
@@ -0,0 +1,33 @@
+ACTIONFILE V4
+
+ENABLED True
+
+INSTALL LongDescription="Simple macros that allows you to create a bookmark named after the system you jumped into by hitting the scroll lock key"
+INSTALL ShortDescription="TGMS Bookmarks"
+INSTALL Version=1.0.0.0
+INSTALL MinEDVersion=8.1.0.0
+INSTALL Location=Actions
+
+EVENT FSDJump, FSDResponse, "", Condition AlwaysTrue
+EVENT onKeyPress, BookmarkSystem, "", KeyPress $== Scroll
+
+//*************************************************************
+// FSDResponse
+// Events: FSDJump
+//*************************************************************
+PROGRAM FSDResponse
+
+Static TGMS_StarSystem = %(EventClass_StarSystem)

```

Comment:
This is here because I couldn't figure out a better way to get the current system name from EDDiscovery. If there's a better way of getting the current system, I'm all ears :D